core/block/import/filetime/android.go

```diff
@@ -0,0 +1,7 @@
+//go:build android
```

Comment:
Do we need a version for `ios` too?

Reply:
Well, I tries to build version for ios, using `make build-ios` and run it in simulator. And it works afau. And for import it doesn't matter, how it's implemented for mobiles, because mobile devices doesn't use ObjectImport request.